assets/js/blocks/single-product/edit/editor.scss

```diff
@@ -0,0 +1,17 @@
+.wc-block-single-product {
+	&.components-placeholder {
+		// Reset the background for the placeholders.
+		background-color: rgba(139, 139, 150, 0.1);
+	}
+
+	.components-resizable-box__handle {
+		z-index: 10;
+	}
+}
+.wc-block-single-product__message {
```

Comment:
And I couldn't find this one either. :confused:

Reply:
I copy pasted from another block - not relevant here.